kive/container/tests_mock.py

```diff
@@ -1048,3 +1051,52 @@ def test_build_dataset_name(self):
             dataset_name = handler.build_dataset_name(run, argument_name)
 
             self.assertEqual(expected_dataset_name, dataset_name)
+
+
+    @patch("container.management.commands.runcontainer.Dataset")
+    @patch("os.rename")
+    @patch("os.walk",
+           return_value=iter([
+               ('/tmp/runsandbox', ['datafiles'], []),
+               ('/tmp/runsandbox/datafiles', ['subdir'], ['a.txt', 'b.txt']),
+               ('/tmp/runsandbox/datafiles/subdir', [], ['c.txt'])
+           ]))
+    def test_save_output_directory(self, mock_walk, mock_rename, dataset_mock):
+        """Simulate saving a directory output called "files/" with the following structure:
```

Comment:
Unless I'm misunderstanding this, it'd actually be `datafiles/`, right?  Or `output`, if we go with Don's suggestion.

Reply:
It would be `datafiles` (which I've fixed), even with Don's suggestion. The mocked directories were different than what Kive sets up.